index.bs

```diff
@@ -794,6 +794,13 @@ To <dfn>queue a report for delivery</dfn> given an [=attribution report=] |repor
     Note: This is intended to allow user agents to optimize device resource usage.
 1. Run [=attempt to deliver a report=] with |report|.
 
+<h3 id="serialize-integer">Serialize an integer</h3>
+
+To <dfn>serialize an integer</dfn>, represent it as a string of the shortest possible decimal number.
+
+Issue: This will be replaced by a more descriptive algorithm in Infra. See
```

Comment:
Given that issue has been open a long time, maybe something like "This would ideally use a..."